test/screenshot/spec/mdc-textfield/classes/invalid-focused-helper-text-persistent-validation-msg.html

```diff
@@ -32,7 +32,7 @@
 
     <div class="test-cell test-cell--textfield">
       <div class="mdc-text-field mdc-text-field--box mdc-text-field--invalid mdc-text-field--focused">
-        <input type="text" id="filled-text-field" class="mdc-text-field__input test-text-field__input">
+        <input type="text" id="filled-text-field" class="mdc-text-field__input test-text-field__input" required pattern=".{4,8}">
```

Comment:
Why do we need both `required` _and_ `pattern`?

Reply:
Required is what forces the asterisk to show. The `pattern` is useful when trying to debug the invalid state using the screenshot tests since you're unable to test the label shake without it.